source/features/stop-redirecting-commit-pull-url-to-commit-url.tsx

```diff
@@ -0,0 +1,31 @@
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../libs/features';
+
+function init(): void {
+	delegate(document, 'form#new_issue', 'submit', updateTextArea);
+	delegate(document, 'form.js-new-comment-form', 'submit', updateTextArea);
+}
+
+function updateTextArea(event: delegate.Event): void {
+	const field = event.delegateTarget.querySelector('textarea');
```

Comment:
If you add `!` here you don't have to add it every time

```suggestion
	const field = event.delegateTarget.querySelector('textarea')!;
```